ios/src/platform_ios.mm

```diff
@@ -54,22 +54,33 @@ bool isContinuousRendering() {
 
 NSString* resolvePath(const char* _path) {
 
-    NSString* path = [NSString stringWithUTF8String:_path];
+    NSString* pathString = [NSString stringWithUTF8String:_path];
+
+    NSURL* resourceFolderUrl = [NSURL fileURLWithPath:[[NSBundle mainBundle] resourcePath]
```

Comment:
I see! If there aren't established practices against it, I would recommend that we just require all scene-related files be located in 'Resources', similar to how we require those files to be in 'assets' on Android.